public/video-ui/styles/components/_keywords.scss

```diff
@@ -0,0 +1,59 @@
+.keywords {
+  border: 1px solid $color500Grey;
+  padding: 10px;
+}
+
+.keywords__item {
+  display: inline-block;
+  color: black;
+  margin-right: 5px;
+  margin-bottom: 5px;
+  padding: 2px 4px;
+
+  border-radius: 3px;
+}
+
+.keyword__item__text {
+  display: inline-block;
+  background-color: $color400Grey;
+  border: 1px solid $color500Grey;
+
+  border-top-left-radius: 3px;
```

Comment:
I prefer the shorthand `border-radius: 0 0 3px 3px` but that's just me,
Also, you /could/ take advantage of Sass here for all the border properties if you like:
```
border: {
    width: 1px;
    color: $color500Grey;
   ...
}
```